src/app/home/home.component.scss

```diff
@@ -0,0 +1,56 @@
+@media (min-width: 768px) {
+    .top-content {
+        width: 64%;
+        margin: 0 auto;
+        text-align: center;
+    }
+
+    .content {
+        width: 64%;
+        margin: 0 auto;
+        text-align: center;
+        img { width: 100%;}
+    }
+
+    h1 {
+        padding-top: 2rem;
+    }
+
+    p {
+        padding-bottom: 2rem;
+    }
+
+    h3 {
+        margin-top: 0rem;
+        text-align: right;
+        color: #B7B7B7;
+    }
```

Comment:
only thing that changes is `.top-content` and `.content` width. you can remove the extras from this block